### PR TITLE
[Communication] - Administration - Fix broken README links

### PR DESCRIPTION
**Pull request by jbeauregardb**

### CODE REVIEW

File: sdk/communication/azure-communication-administration/README.md [hunk 1]
```diff
@@ -189,5 +187,7 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
 <!-- LINKS -->
-[identitysamples]: https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/communication/azure-communication-administration/samples/identity_samples.py
+[identitysamples]: https://github.com/Azure/azure-sdk-for-python/blob/azure-communication-administration_1.0.0b3/sdk/communication/azure-communication-administration/samples/identity_samples.py
 [azure_core]: https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/core/azure-core/README.md
+
+
```
Comment (minnieliu) [inline sdk/communication/azure-communication-administration/README.md:193]: nit: extra lines here

File: sdk/communication/azure-communication-administration/README.md [hunk 2]
```diff
@@ -2,9 +2,7 @@
 
 # Azure Communication Administration Package client library for Python
 
-This package has been deprecated. Please use [azure-communication-identity](https://pypi.org/project/azure-communication-identity/) and [azure-communication-phonenumbers](https://pypi.org/project/azure-communication-phonenumbers/) instead.
-
-The requested features were implemented in the new libraries. See change log for more details.
+Azure Communication Administration client package is intended to be used to setup the basics for opening a way to use Azure Communication Service offerings. This package helps to create identities user tokens to be used by other client packages such as chat, calling, sms.
```
Comment (DominikMe) [inline sdk/communication/azure-communication-administration/README.md:5]: Do we revert this change here on purpose?

Comment (jbeauregardb) [reply sdk/communication/azure-communication-administration/README.md:5]: No, it seems I did the rebase wrong. Correcting it rn

**Review by DominikMe — APPROVED** [review]
